clang/test/3C/multiconstant_bounds.c

```diff
@@ -0,0 +1,19 @@
+// Tests for the 3C.
+//
+// When multiple constants are potential bounds, pick the lower constant.
+// Issue: https://github.com/correctcomputation/checkedc-clang/issues/390
+//
+// RUN: 3c -alltypes %s | FileCheck -match-full-lines %s
+// RUN: 3c -alltypes %s | %clang -c -fcheckedc-extension -x c -o %t1.unused -
+// expected-no-diagnostics
```

Comment:
I think this has no effect unless you pass the `-Xclang -verify` option to `%clang`.  (You could test by expecting a diagnostic and seeing whether the test still passes.)

Reply:
Good catch. Added the flag.